api/src/main/resources/application.conf

```diff
@@ -31,6 +31,10 @@ secrets {
     }
 }
 
-ui {
+api {
```

Comment:
The meat of this change is hidden here, where `API_URL` was pinned to `ui.host` for... reasons. Changing this to be correct (below) is the core fix. Everything else here is tidy up.